tests/Passwordless.AspNetCore.Tests/OldTests/PasswordlessServiceTests.cs

```diff
@@ -196,6 +196,7 @@ public async Task LoginUserAsync_UsesDefaultSchemeIfNoneSpecified()
             _fixture.Create<string>(),
             _fixture.Create<DateTime>(),
             _fixture.Create<Guid>(),
+            _fixture.Create<string>(),
```

Comment:
You can probably replace the default algorithm for the `TestUser` object, which can save you quite a few lines everywhere.

[https://github.com/AutoFixture/AutoFixture/wiki/Cheat-Sheet#replaced-default-algorithm](https://github.com/AutoFixture/AutoFixture/wiki/Cheat-Sheet#replaced-default-algorithm)